app/controllers/follows_controller.rb

```diff
@@ -5,8 +5,9 @@ class FollowsController < ApplicationController
   def create
     if @user and !current_user.follows?(@user)
```

Comment:
Consider renaming `@user` to a name that represents its purpose, some possibilities:

- `@user_to_follow`
- `@followee`